build.gradle

```diff
@@ -15,12 +16,24 @@ buildscript {
 
 allprojects {
     repositories {
+        google()
         jcenter()
         maven { url 'https://jitpack.io' }
 
     }
 }
 
+ext {
+    compileSdkVersion = 25
```

Comment:
If you want to keep offering minSdkVerison of 11 we can't move to 26.

Reply:
Hi jonathan-caryl
Now I don't want :)
minSdkVerison 14 will be fine 👍

Reply:
Hi jonathan-caryl
Now I don't want :)
minSdkVerison 14 will be fine 👍